lib/beaker/host/unix.rb

```diff
@@ -23,5 +23,20 @@ def platform_defaults
       })
     end
 
+    def is_ssh_server_bitvise?
```

Comment:
Why not just return a symbol for the type here?  Less code and more future proof IMHO.

```ruby
def ssh_server_type
  :openssl
end
```